btb_phylo.py

```diff
@@ -269,19 +272,19 @@ def build_multi_fasta(multi_fasta_path, df):
         num_samples = len(df)
         for index, sample in df.iterrows():
             count += 1
-            print(f"Downloading sample: {count} / {num_samples}", end="\r")
+            print(f"adding sample: {count} / {num_samples}", end="\r")
             try:
                 # extract the bucket and key of consensus file from s3 uri
                 s3_bucket = extract_s3_bucket(sample["ResultLoc"])
                 consensus_key = extract_s3_key(sample["ResultLoc"], sample["Sample"])
                 # appends sample's consensus sequence to multifasta
-                append_multi_fasta(s3_bucket, consensus_key, outfile)
+                append_multi_fasta(s3_bucket, consensus_key, outfile, sample, consensus_path)
```

Comment:
I think just passing in the `sample["Sample"]` string instead of the whole row would be a little neater.